onecgiar-pr-client/src/app/shared/components/navigation-bar/navigation-bar.component.ts

```diff
@@ -14,9 +14,24 @@ import { ApiService } from '../../services/api/api.service';
 })
 export class NavigationBarComponent implements OnInit {
   navigationOptions: PrRoute[] = routingApp;
-  emailAccess = ['h.f.tobon@cgiar.org', 'admin@prms.pr', 'j.cadavid@cgiar.org', 'j.delgado@cgiar.org', 'd.casanas@cgiar.org', 'S.Galvez@cgiar.org', 'y.zuniga@cgiar.org', 'yecksin@gmail.com'];
+  emailAccess = [
+    'h.f.tobon@cgiar.org',
+    'admin@prms.pr',
+    'j.cadavid@cgiar.org',
+    'j.delgado@cgiar.org',
+    'd.casanas@cgiar.org',
+    'S.Galvez@cgiar.org',
+    'y.zuniga@cgiar.org',
+    'yecksin@gmail.com'
+  ];
```

Comment:
@yecksin @xKeCo may I know the purpose of these hard-coded emails here? If they are to validate stuff related to admin access, I don't think it is secure to have them written on code

Reply:
You're right, it was already deleted.